docs/src/content/docs/es/guides/site-search.mdx

```diff
@@ -102,13 +104,16 @@ Si tienes acceso al [programa DocSearch de Algolia](https://docsearch.algolia.co
    });
    ```
 
+</Steps>
+
 Con esta configuración actualizada, la barra de búsqueda de tu sitio ahora abrirá un modal de Algolia en lugar del modal de búsqueda predeterminado.
 
 #### Traduciendo la UI de DocSearch
 
 DocSearch solo proporciona cadenas de UI en inglés de forma predeterminada.
 Para agregar traducciones de la UI al modal en otros idiomas, usa el [sistema de internacionalización](/es/guides/i18n/#traduce-la-ui-de-starlight) incorporado de Starlight.
 
+<Steps>
```

Comment:
```suggestion
<Steps>

```

This one will actually also need an empty line to not break when `pnpm format` runs.